docs/Makefile

```diff
@@ -0,0 +1,225 @@
+# Makefile for Sphinx documentation
```

Comment:
Boilerplate generated by `sphinx-quickstart` (keeping it here allows developers to simply run `make html` from the `docs/` dir to test building the docs)